src/main/fc/fc_rc.c

```diff
@@ -259,9 +265,14 @@ FAST_CODE uint8_t processRcInterpolation(void)
 }
 
 #ifdef USE_RC_SMOOTHING_FILTER
-uint8_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline)
+uint16_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline, bool pt1)
 {
-    return lrintf(filterBaseline / (avgRxFrameRate / RC_SMOOTHING_FILTER_AUTO_MS));
+    float cutoff = filterBaseline / (avgRxFrameRate / RC_SMOOTHING_FILTER_AUTO_MS);
```

Comment:
`filterBaseline / (avgRxFrameRate / RC_SMOOTHING_FILTER_AUTO_MS) = filterBaseline * RC_SMOOTHING_FILTER_AUTO_MS / avgRxFrameRate`
filterBaseline is either `RC_SMOOTHING_FILTER_INPUT_AUTO_HZ` or `RC_SMOOTHING_FILTER_DERIVATIVE_AUTO_HZ`, hence it's possible to factor `RC_SMOOTHING_FILTER_AUTO_MS` into both of them. It will make this parameter dimensionless, but I like less constants more.

Reply:
The reason I did it like this was because we determined the default cutoff frequencies based on testing at 9ms frsky SBUS.  So this way it's obvious if we decide that it should be 55hz for example, then we just change the obvious parameter matching our need.  I agree it's a bit clumsy in appearance but it makes it simpler to understand what's going on and adjust.  Combining the two settings (timebase and cutoff frequency) would obscure what we're intending to accomplish.  Perhaps in the future (maybe even through the RC phase) phase we can settle on good final defaults and then we may choose to simplify.  But for now I think it's important to distinguish that we're saying 50hz is the default for 9ms rx frame rates.

Reply:
You can leave 3 parameters, just no need to pass them all into this function